src/commands/communities/addcommunityfilter.js

```diff
@@ -0,0 +1,78 @@
+const { MessageEmbed } = require("discord.js")
+const Command = require("../../base/Command")
+const { getConfirmationMessage } = require("../../utils/responseGetter")
+const { createPagedEmbed } = require("../../utils/functions")
+
+class AddCommunityFilter extends Command {
```

Comment:
There are lots of references to "rule" in this command. I assume this was because you copied from AddRuleFilter but forget to change them.